Documentation/guides/Extending LiveCode.md

```diff
@@ -614,6 +614,54 @@ support/icon.png // 20x40 png image
 support/icon@extra-high.png // 40x80 png image
 ```
 
+## The Extension Store
+
+Once you have created and packaged your extension you can make it available to the LiveCode Community via the Extension Store.
+
+### Registering as an Extension Developer
+
+The first step is to register as an Extension Developer.
+
+- Log in to your LiveCode Account
+- Select the Register option under the Developer section
```

Comment:
@livecodemichael Is it possible to make a link directly to the registration page that redirects via a login page if necessary?